.jenkins/Jenkinsfile-pr

```diff
@@ -202,8 +230,16 @@ pipeline {
     post {
         always {
             script {
+                if(env.TEST_ARCH.contains('s390x')){
+                    sh(script: "minikube stop || true")
+                    sh(script: "minikube delete || true")
+                    sh(script: "docker stop s390x-registry || true")
+                    sh(script: "docker rm s390x-registry || true")
+                }
```

Comment:
Yes, we used a static VM to run the Jenkins pipeline. We could switch to run the pipeline in a docker container so that we do not need these commands.